### PR TITLE
Selenium Tests for LTI Tools

### PR DESCRIPTION
**Pull request by lkiesow**
This patch adds a Selenium based test suite for the LTI tools. They
automatically launch the React app and mock proxy and run against the
mocked data. The tests should usually suffice in determining if e.g. a
library update works.

This patch is based on #2247.

### Your pull request should…

* [x] have a concise title
* [x] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [x] be against the correct branch (features can only go into develop)
* [x] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: modules/lti/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+LTI Tools
```
Comment (gregorydlogan) [inline modules/lti/README.md]: Should we add a requirement for the relevant bits to the development environment setup docs?  Specifically, I do not have the following installed:

- ~selenium~
- geckodriver (or chromedriver)
- npm
Edit: Ah, selenium is a python library which was somehow missed by pip install.

Comment (lkiesow) [reply modules/lti/README.md]: Developers shouldn't usually need to install this.
These tests are not run during the regular build process.
And we are also not listing requirements for other tests.
E.g. you would need `npm` or `yarn` for the admin interface mock or `python`, `mkdocs` and some extensions for building the docs.

It's not that if it is not documented what is required there (and it should be) but it is always documented locally.
Telling developers that they need this when they start Opencast development would just make things more complicated for most people.

**Comment by lkiesow** [comment]
Updated the included #2247

**Comment by lkiesow** [comment]
@pascalseeland, mock data are merged and this is rebased onto the HEAD of `r/9.x`.
=> Ready for a review

**Review by pascalseeland — APPROVED** [review]

**Comment by lkiesow** [comment]
Thanks for the review, @pascalseeland.
@gregorydlogan, since you looked at this as well, do you have any objections to this?

**Comment by gregorydlogan** [comment]
Nope, no problems from my end.

**Comment by lkiesow** [comment]
Then let's merge this.

**Pull request merged**